users/views.py

```diff
@@ -1,3 +1,45 @@
-from django.shortcuts import render
+import json
+import datetime
 
-# Create your views here.
+import jwt
+import bcrypt
+
+from django.http      import JsonResponse
+from django.views     import View
+from django.conf      import settings
+
+from users.models         import User
+from dr_martens.checkitem import CheckItem
+
+class SignInView(View):
+
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+
+            signin_key_list = ["user_id", "password"]
+            CheckItem.check_keys_in_body(data, signin_key_list)
```

Comment:
```suggestion
            data          = json.loads(request.body)
            required_keys = ["user_id", "password"]
            
            CheckItem.check_keys_in_body(data, required_keys)
```
변수명을 다음과 같이 바꾸면 어떨 것 같으세요?

Reply:
리뷰 감사합니다.
signin_key_list를 required_keys로 변경하니, 훨씬 더 직관적인 코드가 된 것 같습니다.
말씀하신 내용 반영하여 수정하였습니다.